tests/publiccloud/prepare_tools.pm

```diff
@@ -83,6 +83,14 @@ sub run {
     assert_script_run("touch .config/ipa/config");
     assert_script_run("ipa list");
     assert_script_run("ipa --version");
+
+    # Download and Install Terraform
+    assert_script_run('wget -q ' . get_required_var('TERRAFORM_URL'));
```

Comment:
should we have a default URL, as we do not configure URLs  for azure-cli nor google-cli?

Reply:
what do you mean "we do not configure URLs for azure-cli nor google-cli" ?  A default URL is also good, hardcoding the latest one.

Reply:
Even I prefer to have as much configuration in code as possible, otherwise the source of information gets diluted

Reply:
done